src/main.cpp

```diff
@@ -521,11 +521,11 @@ static void _assert_not_running() {
 	boost::interprocess::file_lock lock{lock_path};
 	try {
 		if (!lock.try_lock_sharable()) {
-			cerr << "ERROR: aktualizr-lite daemon appears to be running" << endl;
+			cerr << "ERROR: " SOTA_CLIENT " daemon appears to be running" << endl;
```

Comment:
Here we're assuming that the custom client will also use the same lock file (/var/lock/aklite.lock), is this something done/handled by the lite library as well?

Reply:
I think the library should handle this, so the name is fine. That said - the library isn't doing the lock, so i need to work on that logic now also. can be done independently of this.